### DISCUSSION TITLE
大佬，为何你只需要一个ffmpeg.dll就能调用ffmpeg了

### DISCUSSION THREAD

**Discussion by RAVER0306** [wang-bin/mdk-sdk#238] [General]
avdevice-61.dll
avfilter-10.dll
avformat-61.dll
avcodec-61.dll
swscale-8.dll
swresample-5.dll
avutil-59.dll

这些都不需要吗

**Comment by wang-bin** [comment]
内部逻辑，先尝试ffmpeg-7.dll(7为当前编译版本)，不存在则尝试avutil-59.dll这些av模块，还是不存在则尝试无版本号的dll(先ffmpeg.dll, 后avutil.dll），还是失败的话尝试与编译版本最接近的版本（先ffmpeg后av模块）直到不兼容的版本，目前二进制兼容4~7版本。所以你要用那些av dll的话删掉自带的ffmpeg-7.dll就行。单个的ffmpeg dll是为了打包方便。
